libs/sdk-core/src/lsp.rs

```diff
@@ -108,6 +108,23 @@ impl LspAPI for BreezServer {
         Ok(lsp_list)
     }
 
+    async fn list_used_lsps(&self, pubkey: String) -> SdkResult<Vec<LspInformation>> {
```

Comment:
This function name implies the LSP is 'used'. However, the `lsp_full_list` returns any lsp really, also ones that may be unused. Maybe this just needs a rename.

Reply:
I tried to come up with other names, but they were either too long (`list_active_and_historical_lsps`) or not accurate enough (`list_linked_lsps`).

Hopefully the rustdoc in the trait is clear enough about the method's purpose:
```rust
    /// List all LSPs, active and historical, used by the given pubkey
    async fn list_used_lsps(&self, node_pubkey: String) -> SdkResult<Vec<LspInformation>>;
```

If you find a better name, LMK and I'll address it in a follow-up.